packages/vite/src/node/preview.ts

```diff
@@ -45,7 +45,7 @@ export async function preview(
     config.base,
     sirv(distDir, {
       etag: true,
-      dev: !config.isProduction,
+      dev: false,
```

Comment:
> Enable "dev" mode, which disables/skips caching. Instead, sirv will traverse the file system on every request.
Additionally, dev mode will ignore maxAge and immutable as these options generate a production-oriented Cache-Control header value. https://github.com/lukeed/sirv/tree/master/packages/sirv#optsdev

I'm not sure to disable sirv dev mode will be good for DX.  This will cause if the user uses both `yarn build --watch` and `yarn preview` ,  user have to re-run`yarn preview` command to refresh server cache to see the newest result.

Reply:
@patak-js 
If so, then `vite preview` is still in `development` mode by default, but the user is allowed to change the mode by `--mode`，right?

Reply:
`vite preview` should be in production mode by default to fix the issue you mention. I think only call `sirv` with `dev: true` will be ok.